app/interactive_prompt.py

```diff
@@ -152,20 +153,29 @@ def focus(self):
         self.textBuffer.selectionAll()
 
     def formatCommand(self, cmdLine, lines):
-        formatter = {
+        formatters = {
             #".js": app.format_javascript.format
-            #".py": app.format_python.format
             #".html": app.format_html.format,
+            ".py": app.formatter.formatPython
```

Comment:
Just musing (no change needed): hmm, if we have a 'formatter' file (rather than 'format_python' etcetera) maybe this table should move to that file. WDYT (what do you think).
I'm also curious about (though fine with) the decision to go with a 'formatter' file rather than 'format_python'.
(Again, this is all okay for this PR, I'm just interested in your opinion).